pages/cloudflare/caching.mdx

```diff
@@ -16,6 +16,9 @@ The `@opennextjs/cloudflare` caching supports rely on 3 components:
 - A **Queue** to synchronize and deduplicate time-based revalidations
 - A **Tag Cache** for On-demand revalidations via [`revalidateTag`](https://nextjs.org/docs/app/api-reference/functions/revalidateTag) and [`revalidatePath`](https://nextjs.org/docs/app/api-reference/functions/revalidatePath).
 
+You can also enable `cache interception`, which allows to avoid calling `NextServer` and thus loading the javascript associated with the page. It can slightly improve cold start performance for ISR/SSG route.
```

Comment:
```suggestion
You can also enable cache interception, to avoid calling the `NextServer` and thus loading the javascript associated with the page. It can slightly improve cold start performance for ISR/SSG route on cached routes.
```